requirements.txt

```diff
@@ -1,5 +1,5 @@
-ovos_workshop>=0.0.7a8
+ovos_workshop~=0.0.10, >=0.0.7a8
```

Comment:
please use ~=0.0, >=0.0.10
seems to work better when handling requirements with pip installs